<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Главное меню</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_connect">
        <property name="text">
         <string>Подключиться</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_users">
        <property name="text">
         <string>Пользователи</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_update">
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_save">
        <property name="text">
         <string>Сохранить изменения</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Список программ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tableView">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_del_prog">
          <property name="text">
           <string>Удалить программу</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_edit_prog">
          <property name="text">
           <string>Редактировать программу</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_add_prog">
          <property name="text">
           <string>Добавить программу</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
